README.md

```diff
@@ -1,14 +1,60 @@
 # Getting Started with the .NET MAUI Barcode Generator 
-A quick-start project helps you integrate a Barcode Generator control in the .NET MAUI application by adding the Syncfusion MAUI Barcode Generator package from NuGet. This project contains simple code customizations, such as how to generate different Barcode types like QR-Code and Data Matrix.
-
-Refer to the following documentation for the Syncfusion .NET MAUI Barcode Generator Control: 
-https://help.syncfusion.com/maui/barcode-generator/getting-started
-
-Check out this GitHub example for the Syncfusion .NET MAUI Barcode Generator Control: 
-https://github.com/SyncfusionExamples/.NET-MAUI-Barcode-Generator-Getting-Started
+The [.NET MAUI Barcode Generator](https://www.syncfusion.com/maui-controls/maui-barcodes?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples) is a data visualization control that is used to generate and display data in the machine-readable format using industry-standard one-dimensional and two-dimensional barcodes.A quick-start project helps you integrate a Barcode Generator control in the .NET MAUI application by adding the Syncfusion MAUI Barcode Generator package from NuGet. This project contains simple code customizations, such as how to generate different Barcode types like QR-Code and Data Matrix.

```

Comment:
Give space her..
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-barcode-generator/assets/95272441/e8bb508b-cc09-4ceb-b050-35aa4d574dc0)

Reply:
Added space.